src/options.tsx

```diff
@@ -74,7 +74,7 @@ class SvgMapping extends React.PureComponent<SVGIDMappingProps> {
           type="text"
           name="svgId"
           defaultValue={value.svgId}
-          css={config.theme}
+          // css={config.theme}
```

Comment:
Ref: #42 
So commenting this line out actually passes the CI pipeline, but I don't know the original purpoase of `css={config.theme}`.

Reply:
This was injected in commit 131ae6b72da61a87cd5a45b39d094c28c14132de